City-of-Helsinki/drupal-hdbt
# UHF-1760 React and Share layout

How to check with existing setup:

- Switch to corresponding branch:
  - `composer require drupal/hdbt:dev-UHF-1760-react-and-share-layout`
- Run `make drush-cr`
- Go to any page that has React and Share (which is pretty much every page if you have enabled it on your local - if not ask help from Tero)
- Make sure that the React and Share block looks like in the layouts: https://share.goabstract.com/b2f75410-04db-4b86-9295-f9e416b8f997?collectionLayerId=e77d8e05-7cf3-4682-8fcf-5247dfa7458c&mode=build&sha=latest
